crypto/ts/__tests__/Crypto.test.ts

```diff
@@ -195,4 +195,15 @@ describe("Cryptographic operations", function test() {
       expect(commitment).to.eq(commitment2);
     });
   });
+
+  describe("genKeypair", () => {
+    it("should consistently generate a valid keypair (500 runs)", () => {
+      for (let i = 0; i < 500; i += 1) {
+        const key = genKeypair();
+        expect(key.privKey < SNARK_FIELD_SIZE).to.eq(true);
+        expect(key.pubKey[0] < SNARK_FIELD_SIZE).to.eq(true);
+        expect(key.pubKey[1] < SNARK_FIELD_SIZE).to.eq(true);
+      }
+    });
+  });
```

Comment:
I suggest using here https://github.com/dubzzz/fast-check. Property based testing approach will be a good fit.

Reply:
This looks cool, would definitely be useful for other parts of our suites!!!

Reply:
I think I might remove this test case here though, as the issue seems to be with environments !== node 🤔